manifests/install.pp

```diff
@@ -29,4 +29,19 @@
     mode    => '0755',
     source  => 'puppet:///modules/deploynaut/usr/bin/rsync_progress',
   }
+
+  file { "/usr/local/bin/php-resque-staggered-restart.sh":
+    ensure  => file,
+    content => template('deploynaut/php-resque-staggered-restart.sh.erb'),
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0755'
+  }->
+  file { "/etc/cron.d/php-resque-restart":
+      ensure => "file",
+      owner => "root",
+      group => "root",
+      mode => 0644,
+      content => "${restart_cron_minute} ${restart_cron_hour} * * * root /usr/local/bin/php-resque-staggered-restart.sh 2>&1 | logger -t php-resque-restart\n",
```

Comment:
Move this into ldap-password-rotate - this needs to be done straight after that.